plugins/cohere/credentials.go

```diff
@@ -0,0 +1,111 @@
+package cohere
+
+import (
+	"context"
+
+	"encoding/json"
+	"fmt"
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func Credentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.Credentials,
+		DocsURL:       sdk.URL("https://docs.cohere.com/reference/config"),
+		ManagementURL: sdk.URL("https://dashboard.cohere.ai/"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Authtoken,
+				MarkdownDescription: "JWT used to authenticate to Cohere stored in disk.",
+				Secret:              true,
+			},
+			{
+				Name:                fieldname.Email,
+				MarkdownDescription: "Email used to authenticate to Cohere.",
+				Optional:            true,
+			},
+			{
+				Name:                fieldname.URL,
+				MarkdownDescription: "URL of the operator server",
+				Optional:            true,
+			},
+		},
+		DefaultProvisioner: provision.TempFile(
+			cohereJSON,
+			provision.Filename("config"),
+			provision.AtFixedPath("~/.command/")),
```

Comment:
These two options, if I recall correctly, are not both taken into consideration.

I think you're looking for `provision.AtFixedPath("~/.command/config").